src/js/scroller.js

```diff
@@ -15,11 +15,62 @@ RiseVision.Common.Scroller = function (prefs) {
     _items = [],
     _xpos = 0,
     _originalXpos = 0,
+    _customFonts = [],
     _utils = RiseVision.Common.Utilities;
 
   /*
    *  Private Methods
    */
+
+  function getCustomFonts() {
+    for (var i = 0; i < _items.length; i++) {
+      if (_items[i].fontStyle.font.type && (_items[i].fontStyle.font.type === "custom")) {
+        _customFonts.push({
+          "family": _items[i].fontStyle.font.family,
+          "url": _items[i].fontStyle.font.url
+        });
+      }
+    }
+  }
+
+  function loadCustomFonts() {
```

Comment:
@stulees Yes, I think that's a good idea. Same for Google fonts. Shall we add an improvement card?